### ISSUE TITLE
Campaign Options Dialog Has Poor Labels for Accessibility

### ISSUE THREAD

**Issue by BlindGuyNW** [MegaMek/mekhq#2419]
### Environment
What version of MekHQ does your issue apply to? 0.47.17
What operating system are you using?  Windows, but applies across platforms.
What java version are you using?  1.8.0_281, but applies across versions.

### Description

User Impact
Navigating the settings dialog with tab is confusing forscreen reader users because many of the controls lack an association between their label and the control itself. 

Repro steps:
1. Open MekHQ with a screen reader.
2. Start a new campaign and get to the campaign options dialog.
3. Tab across the dialog. The first few controls are for campaign name, faction, and unit rating method, but none of them have labels associated with them.

Actual Results
The fields read as "My Campaign," "Combo box," and "Combo box." This issue is endemic to the whole dialog.

Expected Results
"Name," "Faction," and "Unit Rating method," should be associated with their labels. THis should be accounted for in the redesign and going forward.

Note: See earlier issue #1188, this is a problem across the application.
